src/main/java/net/wurstclient/hacks/FullbrightHack.java

```diff
@@ -46,14 +49,23 @@ public FullbrightHack()
 		
 		EVENTS.add(UpdateListener.class, this);
 	}
-	
+
+	private static double gamma;

```

Comment:
This is not how saving works. A simple field like this gets reset to zero every time the game is restarted...

Reply:
Didn't think on that part, I stopped working with the client, but I think that set the gamma value on startup should fix it, I tested everything, when I changed the gamma in game, it didn't return to previous value, but stayed at it, and always returned to the previous set gamma.